src/components/ToolbarButton.tsx

```diff
@@ -140,10 +142,20 @@ const ToolbarButtonComponent: FC<DraggableToolbarButtonProps> = ({
     [buttonError, colors, dropToRemove, fontSize, isButtonActive, isButtonExecutable, isDragging, longPress.isPressed],
   )
 
+  // deselect button when moving the cursor off the button
+  useEffect(() => {
+    const toolbarButton = toolbarButtonRef.current
+    if (toolbarButton && onMouseLeave) {
+      toolbarButton.addEventListener('mouseleave', onMouseLeave)
+      return () => toolbarButton.removeEventListener('mouseleave', onMouseLeave)
+    }
+  }, [toolbarButtonRef, onMouseLeave])
```

Comment:
Just thought of this, but would it work to just put `onMouseLeave={onMouseLeave}` on the div?

Reply:
That absolutely does work, and this hook is quite silly in hindsight 😅

Refactored: e7a9697ea0fd89408ac29abece6b799381da8c16